source/dokku.rst

```diff
@@ -0,0 +1,222 @@
+Deploying to a Dokku server
+===========================
+
+Projects created using a recent version of the Caktus
+`django-project-template <https://github.com/caktus/django-project-template>`_
+can be deployed to a Dokku server as follows.
+
+These instructions assume
+
+* that ``ssh dokku`` will connect you to your dokku server as the dokku user
+* the `Postgres plugin <https://github.com/dokku/dokku-postgres>`_
+  is installed on the dokku server
+* the `Lets Encrypt plugin <https://github.com/dokku/dokku-letsencrypt>`_
+  is installed on the dokku server
+* an admin has created a directory on the dokku server named
+  ``/var/lib/dokku/data/storage/{{ project_name }}`` that we can use for
+  storage that persists across deploys, and that is writable by the
+  ``dokku`` user.
```

Comment:
Is there anything we could include or point to about this assumption and how to meet it? Even if that is just pointing at the Dokku docs?

Reply:
I'll add a link to the docs.